### DISCUSSION TITLE
Volt Anonymous components :key

### DISCUSSION THREAD

**Discussion by Pennywise96** [livewire/livewire#8043] [1. Help]
I have a deeply nested Volt Anonymous component. When the component renders, I get this error in the console: "Uncaught Snapshot missing on Livewire component with id:".

Is it somehow possible to pass a :key="" to a Volt Anonymous component?

e.g.:
```
@volt("test-component", key(...))
<!-- CONTENT -->
@endvolt
```

It works with a normal Livewire component:

`<livewire:test-component :key="{{ rand() }}" />`

Thanks in advance!

**Comment by joshhanley** [comment]
@Pennywise96 we don't maintain Volt, so I'd suggest asking your question on the Volt repo 🙂
